fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -735,6 +737,10 @@ public static MaterializedViewRewriteMode parse(String str) {
     // this is used to compatible mysql 5.8
     @VariableMgr.VarAttr(name = TRANSACTION_ISOLATION)
     private String transactionIsolation = "REPEATABLE-READ";
+    @VariableMgr.VarAttr(name = TRANSACTION_READ_ONLY)
+    private String transactionReadOnly = "OFF";
+    @VariableMgr.VarAttr(name = TX_READ_ONLY)
+    private String txReadOnly = "OFF";
```

Comment:
can you make them as one field with alias?

Reply:
ok